src/Groups/sub.jl

```diff
@@ -34,7 +34,6 @@ true
 ```
 """
 function sub(G::GAPGroup, gens::AbstractVector{S}; check::Bool = true) where S <: GAPGroupElem
```

Comment:
```suggestion
function sub(G::GAPGroup, gens::AbstractVector{<: GAPGroupElem}; check::Bool = true)
```
since you don't use `S` anymore inside the function

Reply:
o.k.